internal/resource/registry.go

```diff
@@ -58,6 +77,19 @@ func (r *TypeRegistry) Register(registration Registration) {
 		panic(fmt.Sprintf("resource type %s already registered", key))
 	}
 
+	// set default acl hooks for those not provided
+	if registration.ACLs == nil {
+		registration.ACLs = &ACLHooks{}
+	}
+	if registration.ACLs.Read == nil {
+		registration.ACLs.Read = func(authz acl.Authorizer, id *pbresource.ID) error {
+			if err := authz.ToAllowAuthorizer().OperatorReadAllowed(&acl.AuthorizerContext{}); err != nil {
+				return err
+			}
+			return nil
+		}
+	}
+	// TODO: write & list hooks
```

Comment:
Maybe this should be refactored into a `NewRegistration(type, proto, acls, etc)`?